README.md

```diff
@@ -138,6 +138,29 @@ When testing out COVID-Net with PACS integration, be sure to delete any PACS fil
 python3 upload_swift_notify_cube.py
 ```
 
+Currently, the app does not support updating/editing the DICOM images directly. But as an alternative, a connection can be made to the PACS files on Swift, using a database environment (such as datagrip). From there, locate `pacsfiles_pacsfile` and delete the desired DICOMs (which later be updated).
+
+Next, with only the updated DICOM files (that are to update the deleted ones) in `/covidnet_integration/images`, follow the same steps of uploading as above by running the python command above.
+
+#### Creating/Editing DICOM images
```

Comment:
Do you think it might be more suitable to have this script in the `covidnet_integration` repo?

Reply:
Agreed, I'll make the quick change to documentation, so it will be ready once you push the script to covidnet_integration